Gemfile

```diff
@@ -1,9 +1,10 @@
 source 'https://rubygems.org'
 ruby '2.2.2'
 
+gem "csvlint"
+gem "puppet-lint"
+gem "reek"
 gem "rubocop"
+gem "ruby"
+gem "scss-lint", require: false
 gem "sqlint"
-gem 'scss_lint', require: false# require flag is necessary https://github.com/brigade/scss-lint#installation
```

Comment:
does the bear mention the URL https://github.com/brigade/scss-lint#installation ?
if it doesnt, please add a comment in the bear, so that this comment is not lost.